packages/vscode-extension/src/debug/deleteAadHelper.ts

```diff
@@ -84,6 +90,7 @@ export async function deleteAad() {
           localize("teamstoolkit.localDebug.successDeleteNotificationLocalStoreFile")
         );
       }
+      AadSet.clear();
```

Comment:
Is this redundant? `delete` is already called for created aad.
In addition, would this mis-clear the newly created aad since the whole operation is async?